CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # [Unreleased](https://github.com/pybamm-team/PyBaMM/)
 
+# [v25.1.1](https://github.com/pybamm-team/PyBaMM/tree/v25.1.1) - 2025-01-20
+
+## Features
```

Comment:
this is my fault, but there should be a changelog entry about the bugs we fixed.

Reply:
If you want to put a PR up for the bug notice, I will rebase this branch

Reply:
I put one up for this